Assets/HoloToolkit-Examples/InteractiveElements/Scripts/Controls/ButtonThemeWidget.cs

```diff
@@ -30,6 +31,9 @@ public class ButtonThemeWidget : InteractiveThemeWidget
         [Tooltip("scale animation component: optional")]
         public ScaleToValue ScaleSize;
 
+        [Tooltip("Upate the widget's theme and refresh if the theme tag changes")]
+        public bool WatchForThemeChange = false;
```

Comment:
Saving performance is one concern, but is it really saving performance? The real reason is the functionality did not exist before within these existing scripts so I wanted to make sure anything new would not break anything. Having it off by default maintains the existing functionality. Your point is well received though, it's an easy change if you feel the its safe.